src/main/java/use_cases/session_load/SessionLoadUCI.java

```diff
@@ -0,0 +1,48 @@
+package use_cases.session_load;
+
+import entities.SessionStorage;
+import views.file_session_storage.SessionStorageG;
```

Comment:
I just went over Yin's and realized I forgot to check for package imports here so I am going back through yours now. This class should not know that the views package exists, so instead of creating a SessionStorageG object, you should create a Gateway Boundary similar to the Input/Output/View Boundaries that resides in the use_case package, and cast the Gateway to that GatewayBoundary when you are passing it in and storing it in the interaction!

Reply:
Yeah, actually the SessionStorageG was supposed to be that boundary so that the interactor could communicate to the FIleSessionStorage which was the concrete class. But I accidentaly put the SessionStorageG in the worng package. I just moved it inside the use_cases.session_load package and that should be correct now